.babelrc.js

```diff
@@ -0,0 +1,81 @@
+const env = process.env.NODE_ENV;
+const buildIgnore = [
+  '*.jest.js',
+  '*.e2e.js',
+  '*.ssr.js',
+  '*.example.js',
+  'source/demo',
+  'source/jest-*.js',
+  'source/TestUtils.js',
+];
+
+if (env === 'commonjs') {
```

Comment:
personally I just construct a single `module.exports` based on env vars, so i just add plugins, presets (or modify some options) conditionally

that way I better see which plugins are common to all possible values and which ones differ

but whatever works for you :)